src/utilities/api-requests.ts

```diff
@@ -1,4 +1,9 @@
-import axios from "axios";
+import axios, { AxiosError } from "axios";
+
+
+function handleError(uri: string, reason: AxiosError) {
+    console.error(`Request to ${uri} failed with\n${reason}`);
+}
```

Comment:
We might consider replacing Axios with fetch, since fetch is quite ubiquitous in browsers these days.

Reply:
If you think this is valuable, we can do so. What is the advantage of rewriting the code with fetch? Is Axios not well supported?

Reply:
Bandwidth reduction, fewer dependencies with likely vulnerabilities / requiring maintenance (since fetch is a browser API, it updates as you update your browser). We chose Axios at a time when fetch wasn't ready for production use.

Reply:
Fixed.